system/handlers/admin/EditProfile.cfc

```diff
@@ -209,6 +210,40 @@ component output="false" extends="preside.system.base.AdminHandler" {
 		setNextEvent( url = event.buildAdminLink( linkTo="editProfile.twoFactorAuthentication" ) );
 	}
 
+	function resetTwoFactorAuthenticationAction( event, rc, prc ) {
```

Comment:
This should be under `/handlers/admin/UserManager.cfc` (`EditProfile` is for editing your _own_ profile). And there should be a permission check also (same one as you have in the view, below).